zookeeper-server/src/test/java/org/apache/zookeeper/test/WatchEventWhenAutoResetTest.java

```diff
@@ -95,6 +96,7 @@ public void setUp() {
     }
 
     @Test
+    @Ignore("ZOOKEEPER-3182")
```

Comment:
As I mentioned in the Jira, I don't think this is a ZK code problem, but we should improve the reliability in this test case, if the client is disconnected we cannot guarantee it will get the 'expected' watch event without doing sync first.

Reply:
Replied on the jira.